packages/ontario-frontend-cli/core/validation/validatePackageJson.js

```diff
@@ -0,0 +1,42 @@
+const logger = require('../utils/logger');
+const { readPackageJson } = require('../utils/project/readPackageJson');
+
+/**
+ * Checks if a package.json file exists in the given directory.
+ * 
+ * @param {string} dir - The directory to check for package.json.
+ * 
+ * @returns {Promise<boolean>} - Returns a promise that resolves to true if the package.json file exists, false otherwise.
+ */
+async function doesPackageJsonExist(dir = process.cwd()) {
+    try {
+      await readPackageJson(dir);
```

Comment:
Can we simplify this to checking for file existence and privileges, instead of reading the file

```suggestion
      await fs.access(path.join(dir, 'package.json'));
```

Reply:
Sounds good to me.